.cirrus.yml

```diff
@@ -17,6 +17,15 @@ env:
   # GCP_REGION
   # GCP_REPO
 
+# Get the same date for the tasks. CIRRUS_CHANGE_TIMESTAMP is
+# timestamp of when the commit is pushed in ms. Add 500ms to
+# round up to the nearest second then convert it to date. Another method
+# is used at the Windows docker-builder task because $CIRRUS_ENV can't
+# be updated and this method is not working on PowerShell.
```

Comment:
Why can't CIRRUS_ENV be updated? IIRC it was broken at some point, but after I reported that, cirrus implemented a fix?

Reply:
I couldn't update them on Windows, it says: 
'Failed collect CIRRUS_ENV subsystem results: CIRRUS_ENV file should contain lines in KEY=VALUE format'
but it is already in KEY=VALUE format.

[Example task](https://cirrus-ci.com/task/5672164246945792)

Reply:
I suspect that's an encoding issue. Powershell will write utf-16 by default.

Reply:
Ping ^